<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera_settings_dialog</class>
 <widget class="QDialog" name="camera_settings_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Settings</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="settingsLayout" stretch="1,2">
     <item>
      <widget class="QGroupBox" name="gbCamera">
       <property name="title">
        <string>Camera</string>
       </property>
       <layout class="QVBoxLayout" name="camera_layout">
        <item>
         <widget class="QComboBox" name="combo_camera">
          <property name="placeholderText">
           <string>No cameras found</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbSettings">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="settings_layout">
        <item>
         <widget class="QComboBox" name="combo_settings">
          <property name="placeholderText">
           <string>No settings found</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPreview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="preview_layout">
      <item>
       <widget class="QVideoWidget" name="videoWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>camera_settings_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>camera_settings_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
